configs/s2ef/200k/schnet/schnet_steps.yml

```diff
@@ -0,0 +1,27 @@
+includes:
+- configs/s2ef/200k/base.yml
+
+model:
+  name: schnet
+  hidden_channels: 1024
+  num_filters: 256
+  num_interactions: 3
+  num_gaussians: 200
+  cutoff: 6.0
+  use_pbc: True
+
+optim:
+  batch_size: 32
+  eval_batch_size: 16
+  num_workers: 64
+  lr_initial: 0.0005
+  lr_gamma: 0.1
+  lr_milestones: # epochs at which lr_initial <- lr_initial * lr_gamma
+    - 4000
+    - 6000
+    - 8000
+  update_lr_on_step: True
+  warmup_epochs: 2000
```

Comment:
is this warmup_iterations?

Reply:
I'm supporting both step-wise and epoch-wise updates now, as per your earlier suggestion.
If you set `update_lr_on_step` to True, then it's iterations, otherwise, it's epochs.
Let me know if you can think of a better name for this.